src/DragHandle.tsx

```diff
@@ -118,19 +122,26 @@ export default function DragHandle<R, SR>({
     }
   }
 
-  const colSpan = column.colSpan?.({ type: 'ROW', row: rows[rowIdx] }) ?? 1;
-  const style = getCellStyle(column, colSpan);
+  function getStyle(): React.CSSProperties {
+    const colSpan = column.colSpan?.({ type: 'ROW', row: rows[rowIdx] }) ?? 1;
+    const { insetInlineStart, ...style } = getCellStyle(column, colSpan);
+    const marginEnd = 'calc(var(--rdg-drag-handle-size) * -0.5 + 1px)';
+    const isLastColumn = column.idx + colSpan - 1 === maxColIdx;
+
+    return {
+      ...style,
+      gridRowStart,
+      marginInlineEnd: isLastColumn ? undefined : marginEnd,
+      marginBlockEnd: isLastRow ? undefined : marginEnd,
+      insetInlineStart: insetInlineStart
+        ? `calc(${insetInlineStart} + ${columnWidth}px + var(--rdg-drag-handle-size) * -0.5 - 1px)`
+        : undefined
```

Comment:
What does this do exactly?

Reply:
It sets the correct left position for the drag handle when it is frozen.